### PR TITLE
Responsive Desktop Version

### PR DESCRIPTION
**Pull request by sabetech**
In this pull request:
- Top navigation menu shows based on device-width
- Main background image switches dependent on the device-width
- Used media queries to detect device and modify page accordingly

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -170,6 +166,25 @@ a:active {
   margin: 0;
 }
 
+.footer-contents {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 24px 200px 24px;
```
Comment (Amrhub) [inline style.css:173]: - [ ] Let's make sure we get rid of unwanted space at the bottom after footer content (in desktop view) to match the Figma design

Your implementation | Figma Design
:------:|:-----:
image | image

File: style.css [hunk 2]
```diff
@@ -217,6 +243,18 @@ a:active {
   width: 100%;
 }
 
+.inner-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  min-height: calc(100vh - 44px);
+}
```
Comment (Amrhub) [inline style.css:251]: - [ ] Let's make sure in desktop view we don't have these spaces between content to match the Figma design

Your implementation | Figma Design
:------:|:-----:
image | image

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @sabetech,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## To Highlight ✔️ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi @sabetech,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 
giphy

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**